_episodes/05-history.md

```diff
@@ -390,16 +390,19 @@ moving backward and forward in time becomes much easier.
 
 > ## Reverting a Commit
 >
-> Jennifer is collaborating on her Python script with her colleagues and
-> realizes her last commit to the project's repository contained an error and
-> she wants to undo it.  `git revert [erroneous commit ID]` will create a new 
-> commit that reverses Jennifer's erroneous commit. Therefore `git revert` is
-> different to `git checkout [commit ID]` because `git checkout` returns the
-> files within the local repository to a previous state, whereas `git revert`
-> reverses changes committed to the local and project repositories.  
+> Jennifer is collaborating with colleagues on her Python script.  She
+> realizes her last commit to the project's repository contained an error, and 
+> wants to undo it.  Jennifer wants to undo correctly so everyone in the project's
+> repository gets the correct change. The command `git revert [erroneous commit ID]` will create a 
+> new commit that reverses the erroneous commit.  
+> &nbsp;   
```

Comment:
Is there a reason these spacing commands are used at lines 398 and 403? We don't use this formatting elsewhere in the lesson.

Reply:
It was added to improve readability, and make separate paragraphs more distinct.  I'll double-check the readability.

Reply:
@munkm, I removed the nbsp.